File: Notes/SwitchCase.py

count = int(input("Countdown from: "))

match count:
    case 5:
        print("5")
    case 4:
        print("4")
    case 3:
        print("3")
    case 2:
        print("2")
    case 1:
        print("1")
    case _:
        print("Number out of bound.")

File: Notes/SwitchCase.java
import java.util.Scanner;

class SwitchCase {
    public static void main(String[] args) {

        Scanner input = new Scanner(System.in);

        System.out.print("Countdown from: ");
        int count = input.nextInt();

        switch (count){
            case 5:
                System.out.println("5");
            case 4:
                System.out.println("4");
            case 3:
                System.out.println("3");
            case 2:
                System.out.println("2");
            case 1:
                System.out.println("1");
                break;
            default:
                System.out.println("Number out of bound.");
        
        }

        input.close();
    }
}